build_container/build_container_centos.sh

```diff
@@ -27,4 +38,14 @@ chgrp pcap /usr/sbin/tcpdump
 chmod 750 /usr/sbin/tcpdump
 setcap cap_net_raw,cap_net_admin=eip /usr/sbin/tcpdump
 
+# The build_container_common.sh will be skipped when building centOS
+# image on Arm64 platform since some building issues are still unsolved. 
+# It will be fixed until those issues solved on Arm64 platform.
+if grep -q -e rhel /etc/*-release; then
```

Comment:
You don't need this `if`, since this file will be only run under CentOS.

Reply:
Has been solved. Thanks.